conf/rosalind.config

```diff
@@ -0,0 +1,28 @@
+params {
+	config_profile_description = 'Kings College London Rosalind HPC'
```

Comment:
I think we have used 2 space tabs everywhere else. Be good to keep it consistent :)

Reply:
Sure thing, I will make those changes now :)